src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -23,9 +24,14 @@ const ORDER_STATUS_APPROVED = 'APPROVED';
 const ORDER_STATUS_CREATED = 'CREATED';
 const POLLING_INTERVAL = 3000;
 const POLLING_MAX_TIME = 600000;
+const LOADING_INDICATOR_CONTAINER = 'checkout-app';
```

Comment:
The container ID should not be hard-coded. Can you find a way to pass it in?

Reply:
fixed